src/main/org/pantsbuild/jarjar/util/StandaloneJarProcessor.java

```diff
@@ -52,7 +52,7 @@ public static void run(File from, File to, JarProcessor proc) throws IOException
                     } else if (struct.name.endsWith("/")) {
                         // TODO(chrisn): log
                     } else {
-                        throw new IllegalArgumentException("Duplicate jar entries: " + struct.name);
+                        throw new DuplicateJarEntryException(from.getAbsolutePath(), struct.name);
```

Comment:
I like making it an explicit exception, but I feel like an IllegalArgumentException probably should return a non zero exit status to the shell too.  This is our copy of jarjar, we can do what we want.